docs/contributor/performance-tests/metrics/metrics.md

```diff
@@ -0,0 +1,690 @@
+- Metric count: 500
+- Metric series: 20
+- Number of labels: 10
+
+To simulate a smooth ramp-up and avoid huge data flood at the beginning, load generator was started with 5 instances, instance count was increased at 5 instances every minute until peak was reached.
+
+This test was executed on a single node installation to determine the limits of a single metric agent, after the load generator reached `28` instances, metric agent hits the limits and stays stable with this setup. Any load above this setup would cause memory limiter to refuse incoming traffic.
+
+The graph below shows the average metric points processed by the `prometheus metric receiver` per second, agent reaches an average of `14K metric points/sec`
+![Peak accepted metric points](./assets/overall-peak-metric-point.jpg)
+
+Following graph shows the refused metric points from the configured `memory_limiter`, with traffic above 14K metric points/sec results in  metrics being refused by the `memory_limiter`. 
+![Peak refused metric points](./assets/overall-peak-metric-point-refused.jpg)
+
+Following graph shows memory utilization of metric agent during test phase.
+![Peak memory utilization of metric agent](./assets/overall-peak-memory.jpg)
+
+
+Following graph shows CPU utilization of metric agent during test phase.
+![Peak CPU utilization of metric agent](./assets/overall-peak-cpu.jpg)
+
+
+### Workload generating huge amount of metrics
+
+This scenario tests a single metric endpoint with huge amount of data, Avalanche load generator deployed with a single pod instance and configured with 1000 distinct metrics and 20 metric series for each metric with 10 labels.
+
+Setup:
+Avalanche load generator configuration:
+- Metric Count: 1000
+- Metric Series: 20
+- Number of label: 10
+
+Metric count was increased in 1000 steps to find the limit of the metric agent.
+
+Following graph shows max stable load from single target with huge amount of metric data.
+![Peak accepted metric points](./assets/single-target-peak-metric-point.jpg)
+
+Metric agent reached max `7K metric points/sec` from a single metric target, with 6000 and 20 metric series. In this test scenario the 
+ measured raw metric payload size was `~51MB`
+
+Another test was performed to determine the impact of different metric series on metric agent. For this the test metric series were increased from `20` to `30`, distinct metric count started from `1000` and increased in `1000` steps until the limit of metric agent was reached.
+Metric agent reached a peak of `~7K metric points/sec` (yellow line from graph above) with `4000` distinct metric. In this setup the measured raw metric payload size was `~31MB`
+
+In both the configurations metric agent reached a peak of `~7K metric points/sec` independent size of raw metric data, thus raw metric payload size has no impact on metric agent performance.
+
+In both configurations Avalanche load generator generated `120K metric points/scrape` 
```

Comment:
```suggestion
In both configurations, the Avalanche load generator generated `120K metric points/scrape` .
```
Same here, why are some elements formatted in code font? Does any of our [guidelines](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/03-formatting.md#use-code-font-for-these-items) apply?